packages/expo-notifications/README.md

```diff
@@ -1,5 +1,115 @@
 # expo-notifications
 
-`expo-notifications` provides some great stuff.
+## Instalation 
+
+`expo-notifications` needs `react-native-unimodule` to work. You can read more about it [here](https://www.npmjs.com/package/react-native-unimodules). 
+
+Add `expo-notifications` to your project:
+execute the following command in your project root directory:
+"yarn add expo-notifications".
+
+### Android
+
+#### 1. Modify AndroidManifest
+
+`AndroidManifest.xml` is a place where you can configure `expo-notifications` module. For Instance, you can choose there
+if you want to use Expo's servers as a middleman or choose that you don't want to use push notifications at all.
+
+Configuration template:
+
+```xml
+
+<!-- expo-notifications START -->
+    <!-- turn on push notifications START -->
+        <service
+        android:name="expo.modules.notifications.push.fcm.ExpoFcmMessagingService"
+        android:exported="false">
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT" />
+            </intent-filter>
+        </service> 
+
+        <!-- choose if you want to use expo servers START -->
+        <meta-data android:name="appId" android:value="@username/slug" /> <!-- only with expo engine -->
+        <meta-data android:name="pushNotificationEngine" android:value="expo" /> <!-- expo|bare -->
+        <!-- choose if you want to use expo servers END -->
+
+    <!-- turn on push notifications END -->
+
+    <!-- tell us which activity should be launched when notification is tapped START -->
+        <meta-data android:name="notificationReceiver" android:value="com.example.MainActivity" />
```

Comment:
Is it possible to infer the `MainActivity` from `intent-filter`: `.DEFAULT` or `.MAIN` or `.LAUNCHER`?

This should simplify the setup and, most importantly, work for 85% of the users. Advanced users could configure the value following some `Advanced customizations` guides.

Reply:
Or maybe we can define our own action and ask users to add it to Activity definition in Manifest?